config/deploy.rb

```diff
@@ -14,3 +13,13 @@
 set :linked_dirs, fetch(:linked_dirs, []).concat(
   %w(bin log tmp/pids tmp/cache tmp/sockets vendor/bundle public/system public/files),
 )
+
+namespace :eye do
+  task :set_recurring_tasks do
+    on roles :all do |host|
+      set :eye_env, -> { { rails_env: fetch(:rails_env), recurring: host.roles_array.include?(:db) } }
```

Comment:
why do this complicated thing, when line 19 seems to allow for `on roles :db do |host|`, no?